python/ml4ir/applications/ranking/model/metrics/helpers/metrics_helper.py

```diff
@@ -0,0 +1,165 @@
+from typing import List
+
+import pandas as pd
+
+from ml4ir.applications.ranking.model.metrics.helpers.metric_key import Metric
+from ml4ir.applications.ranking.model.metrics.helpers.aux_metrics_helper import compute_aux_metrics_on_query_group
+
+DELTA = 1e-20
+
+
+def get_grouped_stats(
```

Comment:
@mohazahran This is the same old file too. I just moved it all under a `helpers/` subdirectory and split it up into `metrics_helper` and `aux_metrics_helper` as the file was becoming too big.